.github/workflows/sca.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 env:
   environment: "qa"
-  timezone: "Europe/London"
+  TIMEZONE: ${{ vars.TIMEZONE }}
```

Comment:
I think it would be nicer if the casing was consistent - personally, if I see inconsistent casing, it makes me think that something is case _sensitive_, which GHA variables aren't unless I'm mistaken.